autogpt_platform/backend/backend/data/block.py

```diff
@@ -53,6 +53,7 @@ class BlockCategory(Enum):
     COMMUNICATION = "Block that interacts with communication platforms."
     DEVELOPER_TOOLS = "Developer tools such as GitHub blocks."
     DATA = "Block that interacts with structured data."
+    HARDWARE = "Block that interacts with hardware."
```

Comment:
`ROBOTICS`?

Hardware is very very broad.

Reply:
its not technically robotics though

Reply:
It being called ``HARDWARE`` i feel is a good choice, it is broad but if we where to break it down more it will probably start going to actual component names like Raspberry Pi, ESP ect which i feel would be taking this to far? so keeping it as generic ``HARDWARE`` would it better to fit and i feel would work better in the longer run?